### PR TITLE
clean up js

### PR DESCRIPTION
**Pull request by MagdaN**

### CODE REVIEW

**Comment by rmader** [comment]
Is this still WIP?

**Comment by MagdaN** [comment]
yes

File: meinberlin/apps/maps/assets/map_choose_polygon_with_preset.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 /* global django */
 import { createMap } from 'a4maps_common'
 import 'leaflet-draw'
-import '../../../assets/js/i18n-leaflet-draw'
+import '../assets/i18n-leaflet-draw'
```
Comment (rmader) [inline meinberlin/apps/maps/assets/map_choose_polygon_with_preset.js]: Could be `'./i18n-leaflet-draw'`

File: meinberlin/templates/a4dashboard/base_dashboard_project.html [hunk 2]
```diff
@@ -187,8 +187,3 @@ <h2 class="modal-title u-first-heading" id="delete-modal-label">{% trans "Are yo
         </div>
     </div>
 {% endblock %}
-
-
-{% block extra_js %}
-    <script src="{% static 'js/unload_warning.js' %}"></script>
```
Comment (rmader) [inline meinberlin/templates/a4dashboard/base_dashboard_project.html]: While I do agree with this change here, it's a bit odd to load one big monolithic js file everywhere. But for our current webpack setup it's probably the best approach :/

Comment (rmader) [reply meinberlin/templates/a4dashboard/base_dashboard_project.html]: Some for the ajax modal btw.

File: meinberlin/apps/maps/assets/map_choose_polygon_with_preset.js (hunk 1 above)
Comment (MagdaN) [reply meinberlin/apps/maps/assets/map_choose_polygon_with_preset.js]: true

File: meinberlin/templates/a4dashboard/base_dashboard_project.html (hunk 2 above)
Comment (MagdaN) [reply meinberlin/templates/a4dashboard/base_dashboard_project.html]: I agree, but we should come up with a generell rule (maybe we could have webpack bundles for each app) here, and the unload_warning actually makes sense for all forms I would say the ajax_modal not so much.

**Pull request merged**